README.md

```diff
@@ -4,22 +4,56 @@ This is an online multiplayer take on the classic party game Mafia. The game spl
 However, since this is an OSS project, developers are free to edit rules and add new functionality as they please.
 
 <p align="center">
-<img src="https://cdn2.iconfinder.com/data/icons/rounded-white-basic-ui-set-3/139/Photo_Add-RoundedWhite-512.png" height=200; width=200>
+  <img src="https://cdn2.iconfinder.com/data/icons/rounded-white-basic-ui-set-3/139/Photo_Add-RoundedWhite-512.png" height=200; width=200>
 </p>
 
 
 Please review the [documentation](https://github.com/Mafia-Hands/Mafia/wiki) for guidelines and information about the software.
 
 ## Installation
+Before you install & get started with the Mafia project, please you have installed [nodejs](https://nodejs.org/en/) and [npm](https://www.npmjs.com/) on your computer.
 
-Use the package manager [npm](https://www.npmjs.com/get-npm) to install all dependencies.
+This project uses the package manager [npm](https://www.npmjs.com/get-npm) to install all dependencies.
 
+### For frontend
 ```bash
+git clone git@github.com:Mafia-Hands/Mafia.git
+cd Mafia
+cd frontend
 npm install
+npm start
 ```
+
+### For backend
+```bash
+git clone git@github.com:Mafia-Hands/Mafia.git
+cd Mafia
+cd backend
+npm install
+npm start
+```
```

Comment:
This looks quite confusing. It will be better to lay it out like so:

First, clone the repository and open the project directory:
```
git clone git@github.com:Mafia-Hands/Mafia.git
cd Mafia
```

To run this application locally, the frontend and backend must run together. You can do it like so:

Frontend:
```
cd frontend
npm install
npm start
```
Backend:
```
cd backend
npm install
npm start
```
**Note:** You do not need to run `npm install` everytime. Only when a new package is added.
**Note:** Ensure that the `socket.js` file (frontend > src > socket.js) points to your local server by uncommenting the necessary line.